lab-anthony/lib/parse-json.js

```diff
@@ -0,0 +1,18 @@
+'use stract';
+
+module.exports = function(req) {
+  return new Promise(function(resolve, reject) {
+    req.body = '';
+    req.on('data', function(data) {
+      req.body += data.toString();
+    });
+    req.on('end', function() {
+      try {
+        req.body = JSON.parse(req.body);
+        resolve();
```

Comment:
If you're calling reject with an error, call resolve with null to make it clear that there wasn't an error.